spec/functional/manage_controller_spec.rb

```diff
@@ -135,6 +135,12 @@ module Pod::TrunkApp
     end
 
     it 'updates a dispute' do
+      put "/disputes/#{@disputes.first.id}", :dispute => { :message => 'HELLO' }
+      last_response.status.should == 302
+      @disputes.first.reload.message.should == 'HELLO'
+    end
+
+    it 'sends slack notification for a dispute' do
```

Comment:
couldn't think of a way to verify that the slack message was triggered, we didn't have them for the other one either